### DISCUSSION TITLE
marmite/enabling-comments

### DISCUSSION THREAD

**Discussion by giscus** [rochacbruno/marmite#68] [Comments]
# marmite/enabling-comments

https://rochacbruno.github.io/marmite/enabling-comments.html

**Comment by rochacbruno** [comment]
Hello World! 

We have comments!!!

**Comment by lobovit** [comment]
Hey @rochacbruno  I found something that worries me. The Yaml Rust Library we use seems to not have a maintener. Look: https://github.com/chyh1990/yaml-rust/issues/197 . Consider switching to the actively maintained `yaml-rust2` fork of the original project https://github.com/Ethiraric/yaml-rust2

**Reply by rochacbruno** [reply]
It looks like this lib is a dependency of another lib.

```
syntect: # < this installs the outdated yaml-rust
   comrak:
      marmite:
```

**Reply by lobovit** [reply]
Ah, ok :) https://github.com/trishume/syntect/issues/537

**Comment by ghost** [comment]
Muito massa Bruno (vou ver como configurar o Hatsu)!

**Comment by bilalijaz662-creator** [comment]
"Best game to show your creativity! " <a href="https://www.minecraft.net/en-us">minecraft</a>
